### PR TITLE
Blog App : Controllers

### PR DESCRIPTION
**Pull request by torobucci**
-  For each URL displayed in diagram create:
    - A route.
    - An action in the correct controller.
    - A view file.
-  Ensure view file has a HTML placeholder

### CODE REVIEW

**Review by prolajumokeoni — APPROVED** [review]
Hi @torobucci, You have submitted a project, that does not have the complete [linter setup](https://github.com/microverseinc/linters-config/tree/master/ror), please add the stylelint configuration from the microverse linter config👍🏿

**Review by prolajumokeoni — COMMENTED** [review]
Hi @torobucci 


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the complete [linter setup](https://github.com/microverseinc/linters-config/tree/master/ror), please add the stylelint configuration from the microverse linter config👍🏿 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show', as: 'user'
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```
Comment (AmaduKamara) [inline config/routes.rb]: ## Suggestion:

- [x] You could implement this in a much better way as this:
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @torobucci,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Created controllers and views files correctly 👍🏽
✅ Created routes paths correctly 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**